doc/CMakeLists.txt

```diff
@@ -86,6 +88,9 @@ if(DOXYGEN_FOUND)
     set(DOXYGEN_USE_MDFILE_AS_MAINPAGE "../README.md")
     set(DOXYGEN_EXCLUDE_PATTERNS "*/node_modules/*")
     set(DOXYGEN_IMAGE_PATH "./")
+    set(DOXYGEN_MARKDOWN_ID_STYLE GITHUB)
+    set(DOXYGEN_DOT_IMAGE_FORMAT svg)
+    set(DOXYGEN_INTERACTIVE_SVG YES)
```

Comment:
I was hoping they'd do something. Since the dependency graph is an SVG, it would be nice to get the pan-and-zoom features when embedding it. But it seems like the setting only applies to DOT charts and not to other SVGs.